datauri/datauri.py

```diff
@@ -22,6 +22,18 @@ class DataURIError(ValueError):
     pass
 
 
+# https://github.com/bottlepy/bottle/commit/fa7733e075da0d790d809aa3d2f53071897e6f76
+class CachedProperty(object):
```

Comment:
nit: `cached_property` (or an alias `cached_property = CachedProperty`) makes usage look more like `@property` for which this is a drop-in replacement anyway

Reply:
Fixed :+1: